libs/langchain/langchain/llms/huggingface_hub.py

```diff
@@ -107,15 +108,17 @@ def _call(
         """
         _model_kwargs = self.model_kwargs or {}
         params = {**_model_kwargs, **kwargs}
-        response = self.client(inputs=prompt, params=params)
+        response = self.client.post(
+            json={"inputs": prompt, "params": params}, task=self.task
+        ).json()
```

Comment:
okay, I will review it and fix it.

Reply:
@easelify @baskaryan I again the ran the following code:
```python

from langchain.llms import HuggingFaceHub
from langchain.prompts import PromptTemplate
from langchain.chains import LLMChain
import os
os.environ['HUGGINGFACEHUB_API_TOKEN'] = ''
prompt = PromptTemplate(template = 'Please answer the following question: {question}',input_variables=['question'])

llm_chain = LLMChain(prompt=prompt,
                     llm=HuggingFaceHub(
                         repo_id="google/flan-t5-small",
                         model_kwargs={
                             "temperature": 0,
                             "max_length": 64
                         }
                     )
                     )

question = "What is the capital of England?"

print(llm_chain.run(question))

```

and got no errors. The `InferenceClient` post method returns a Python Request object which is converted to json object therefore I don't get it why you guys are getting a bytes object. It might be due to different version for huggingface_hub.

Can I ask which version of `huggingface_hub` are you guys using? My version is as follows
```
ame: huggingface-hub
Version: 0.15.1
Summary: Client library to download and publish models, datasets and other repos on the huggingface.co hub
Home-page: https://github.com/huggingface/huggingface_hub
Author: Hugging Face, Inc.
Author-email: julien@huggingface.co
License: Apache

```

Reply:
@easelify ???

Reply:
@easelify waiting for your reply.

Reply:
Sorry, I have been offline for a while recently. The version I am using is 0.17.2